lib/api/core/entrypoints/embedded/protocols/http.js

```diff
@@ -56,223 +71,228 @@ class HttpProtocol extends Protocol {
    * @param {EmbeddedEntryPoint} entryPoint
    */
   init(entryPoint) {
-    super.init(entryPoint);
-
-    debug('initializing http Server with config: %a', entryPoint.config);
+    return super.init('http', entryPoint)
+      .then(() => {
+        const config = entryPoint.config.protocols.http;
```

Comment:
Why not storing the config for every protocol ? You need it here for example: https://github.com/kuzzleio/kuzzle/pull/1301/files#diff-77ed97acf7a889e5f1c8143d237974feR403
Some protocol save the config object (mqtt) and not the other ones, this is inconsistent
 
This can be done in the `Protocol.init` methods for example